scss/_patterns_navigation.scss

```diff
@@ -1,8 +1,8 @@
 @import 'settings';
 
 $nav-border-bottom-thickness: 1px;
+$hover-opacity: 0.58;
```

Comment:
Can these variables be scoped to the navigation pattern

Reply:
The border thickness is only used in one place and there are other hard-coded sizes based on it. I'll remove the variable entirely.

The opacity can't be scoped because it's used across mixins but I'll name it unambiguously